app/views/services/index.html.erb

```diff
@@ -4,8 +4,8 @@
   <table class="govuk-table">
     <thead class="govuk-table__head">
       <tr class="govuk-table__row">
-        <th class="govuk-table__header" scope="col">Name</th>
-        <th class="govuk-table__header" scope="col">Entity ID</th>
+        <th class="govuk-table__header" scope="col"><%= t 'certificates.header_name' %></th><!-- should i be reusing this -->
```

Comment:
Is this okay to be reused? Doesnt feel right to have a reference to certificates or msa_component on this view, so should the repeated heading be moved under a more general heading or is this acceptable?